openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -22,12 +22,23 @@
 
 def get_profile_image_storage():
     """
-    Configures and returns a django Storage instance that can be used
-    to physically locate, read and write profile images.
+        Returns a configured Django Storage instance for handling profile images.
+        The storage backend is defined in the Django setting `PROFILE_IMAGE_BACKEND`,
+        which should be a dictionary with the following structure:
+            {
+                'class': 'full.path.to.StorageClass',
+                'options': {
+                    # Optional keyword arguments passed to the storage class constructor
+                }
+            }
+
+        This function dynamically loads the specified storage class and initializes it
+        with the provided options.
     """
     config = settings.PROFILE_IMAGE_BACKEND
-    storage_class = get_storage_class(config['class'])
-    return storage_class(**config['options'])
+    storage_class_path = config.get('class')
+    storage_class = import_string(storage_class_path)
+    return storage_class(**config.get('options', {}))
```

Comment:
Why are not we using the below one?

```
from django.core.files.storage import storages
storage_class = storages(config.get('class'))
return storage_class(**config.get('options', {}))
```

Reply:
https://github.com/carltongibson/django/blob/5b1099323288b15a4c23cba88eb4dcdfa824b3d4/django/core/files/storage/__init__.py#L37C5-L37C71